cli/tests/pcluster/templates/test_cluster_stack/test_pcluster_dashboard_builder/dashboard_config_1.yaml

```diff
@@ -0,0 +1,86 @@
+Image:
+  Os: centos8
+HeadNode:
+  InstanceType: t2.micro
+  Networking:
+    SubnetId: subnet-12345678
+  Ssh:
+    KeyName: ec2-key-name
+  Dcv:
+    Enabled: true
+    Port: 8443
+    AllowedIps: 0.0.0.0/0
+Scheduling:
+  Slurm:
+    Queues:
+      - Name: queue1
+        Networking:
+          SubnetIds:
+            - subnet-12345678
+        ComputeResources:
+          - Name: compute_resource1
+            InstanceType: c5.2xlarge
+          - Name: compute_resource2
+            InstanceType: c4.2xlarge
+      - Name: queue2
+        Networking:
+          SubnetIds:
+            - subnet-12345678
+        ComputeResources:
+          - Name: compute_resource1
+            InstanceType: c5.2xlarge
+            MaxCount: 5
+          - Name: compute_resource2
+            InstanceType: c4.2xlarge
+SharedStorage:
+  - MountDir: /my/mount/ebs1
+    Ebs:
+      VolumeType: gp2  # gp2 | gp3 | io1 | io2 | sc1 | st1 | standard
```

Comment:
nit: you can remove the comments, they were there just to describe the defaults or the alternatives but they are not required in the test file.